sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -206,9 +210,107 @@ public partial class RegistryArtifact
+        public System.IO.Stream Content { get { throw null; } }
+        public string Digest { get { throw null; } }
+        public void Dispose() { }
+    }
+    public partial class DownloadManifestOptions
+    {
+        public DownloadManifestOptions(string tag = null, string digest = null) { }
+        public string Digest { get { throw null; } }
+        public string Tag { get { throw null; } }
+    }
+    public partial class DownloadManifestResult : System.IDisposable
+    {
+        internal DownloadManifestResult() { }
+        public string Digest { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.OciManifest Manifest { get { throw null; } }
+        public System.IO.Stream ManifestStream { get { throw null; } }
+        public void Dispose() { }
+    }
+    public partial class OciAnnotations
+    {
+        public OciAnnotations() { }
+        public System.Collections.Generic.IDictionary<string, object> AdditionalProperties { get { throw null; } }
+        public string Authors { get { throw null; } set { } }
+        public System.DateTimeOffset? CreatedOn { get { throw null; } set { } }
+        public string Description { get { throw null; } set { } }
+        public string Documentation { get { throw null; } set { } }
+        public string Licenses { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public string Revision { get { throw null; } set { } }
+        public string Source { get { throw null; } set { } }
+        public string Title { get { throw null; } set { } }
+        public string Url { get { throw null; } set { } }
+        public string Vendor { get { throw null; } set { } }
+        public string Version { get { throw null; } set { } }
+    }
+    public partial class OciBlobDescriptor
+    {
+        public OciBlobDescriptor() { }
+        public Azure.Containers.ContainerRegistry.Specialized.OciAnnotations Annotations { get { throw null; } set { } }
+        public string Digest { get { throw null; } set { } }
+        public string MediaType { get { throw null; } set { } }
+        public long? Size { get { throw null; } set { } }
+    }
+    public partial class OciManifest
+    {
+        public OciManifest() { }
+        public Azure.Containers.ContainerRegistry.Specialized.OciAnnotations Annotations { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.OciBlobDescriptor Config { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.Containers.ContainerRegistry.Specialized.OciBlobDescriptor> Layers { get { throw null; } }
+        public int? SchemaVersion { get { throw null; } set { } }
+    }
+    public partial class UploadBlobResult
+    {
+        internal UploadBlobResult() { }
+        public string Digest { get { throw null; } }
+    }
+    public partial class UploadManifestOptions
+    {
+        public UploadManifestOptions() { }
```

Comment:
Would we want to make this take an optional tag in the constructor to be consistent with DownloadManifestOptions?

Reply:
Done.